Gemfile

```diff
@@ -56,6 +56,7 @@ gem "inifile",                        "~>3.0",         :require => false
 gem "jbuilder",                       "~>2.5.0" # For the REST API
 gem "manageiq-api-client",            "~>0.1.0",       :require => false
 gem "mime-types",                     "~>2.6.1",       :require => "mime/types/columnar"
+gem "money-rails",                    "~>1"
```

Comment:
Can the places that use it do the require?  We're trying to avoid requiring everything just booting the rails app.  I'm unsure where money-rails is used.  Autoloading it in `config/initializers/money.rb` is problematic because we're trying to extract dependencies to the code that uses it.  To be honest, I can't even tell what in this PR uses MoneyRails besides the initializer.  We should make it more obvious what's using it.